docs/contracts/uniswapx/guides/priorityorderreactor.md

```diff
@@ -11,12 +11,49 @@ Alice submits a PriorityOrder offering 1 ETH in exchange for a minimum of 1000 U
+- Only the fill transaction with the highest priority fee will win the order, all other transactions will revert on chain.
+- To minimize the gas used on reverting transactions, we revert early if the order is already filled or is not fillable yet.
+- For every wei of priority fee above a certain threshold (an optional value specified in the order), the user is owed 1 milli-bps more of their output token (or less of their input token). 
+- Milli-bps (or MPS) are one-thousandth of a basis point.
+
+## Retrieving and Executing Signed Orders
+All signed Priority Orders created through the Uniswap UI will be available via the UniswapX Orders Endpoint. We have [swagger documentation](https://api.uniswap.org/v2/uniswapx/docs) but see below for a quick example curl.
+
+```
+GET https://api.uniswap.org/v2/orders?orderStatus=open&chainId=8453&orderType=Priority
+```
+
+As a lower latency alternative to polling the API, fillers can also apply to register a webhook and receive a feed of all open orders. See details for registering [here](./webhooks). 
+
+After fetching orders, use the latest version of the [UniswapX SDK](https://github.com/Uniswap/sdks/tree/main/sdks/uniswapx-sdk#parsing-orders). Requires [2.1.0-beta.13](https://www.npmjs.com/package/@uniswap/uniswapx-sdk/v/2.1.0-beta.13) or later.
+
+### Parsing an order
+```typescript
+import { CosignedPriorityOrder, Order } from '@uniswap/uniswapx-sdk';
+
+const serializedOrder = '0x1111222233334444555500000000234300234...';
+const chainId = 1; 
+
+const order: Order = CosignedPriorityOrder.parse(serializedOrder, chainId);
+
+const orderData = order.info;
+const orderHash = order.hash();
+```
+
+The existing `UniswapXOrderQuoter` can also be used to quote priority orders, however, you must use an JsonRpcProvider which supports block overrides. Without block overrides, the SDK quoter cannot validate the entire order as the block number is checked first in the contract.
+
+## Executing an order
+The [PriorityOrderReactor](https://github.com/Uniswap/UniswapX/blob/main/src/reactors/PriorityOrderReactor.sol) shares the same interface as all other existing UniswapX reactors. Orders are executed against the `execute` and `executeBatch` functions, and optionally a callback is available via `executeWithCallback` and `executeBatchWithCallback`.
+
+# Deployment addresses
+The PriorityOrderReactor is deployed on the following chains:
+| Chain                      | Address                                                                                                               | Source                                                                                                                    |
```

Comment:
broke in preview
<img width="945" alt="Priority_Order_Reactor___Uniswap" src="https://github.com/user-attachments/assets/dbbc19db-e4ef-4240-9098-68a1a35a24f5">